lib/ex_ice/priv/ice_agent.ex

```diff
@@ -1647,6 +1637,66 @@ defmodule ExICE.Priv.ICEAgent do
     ice_agent
   end
 
+  defp handle_keepalive_response(
+         ice_agent,
+         local_cand,
+         src_ip,
+         src_port,
+         %Message{type: %Type{class: :success_response}} = msg
+       ) do
+    {pair_id, ice_agent} = pop_in(ice_agent.keepalives[msg.transaction_id])
+    pair = Map.fetch!(ice_agent.checklist, pair_id)
+
+    with true <- symmetric?(ice_agent, local_cand.base.socket, {src_ip, src_port}, pair),
+         :ok <- authenticate_msg(msg, ice_agent.remote_pwd) do
+      Logger.debug("Received keepalive success response on: #{pair_info(ice_agent, pair)}")
+      pair = %CandidatePair{pair | last_seen: now()}
+      put_in(ice_agent.checklist[pair.id], pair)
+    else
+      false ->
+        ka_local_cand = Map.fetch!(ice_agent.local_cands, pair.local_cand_id)
+        ka_remote_cand = Map.fetch!(ice_agent.remote_cands, pair.remote_cand_id)
+
+        Logger.warning("""
+        Ignoring keepalive success response, non-symmetric src and dst addresses.
+        Sent from: #{inspect({ka_local_cand.base.base_address, ka_local_cand.base.base_port})}, \
+        to: #{inspect({ka_remote_cand.address, ka_remote_cand.port})}
+        Recv from: #{inspect({src_ip, src_port})}, on: #{inspect({local_cand.base.base_address, local_cand.base.base_port})} \
+        Not refreshing last_seen time. \
+        """)
```

Comment:
Why is this a warning log? Is it more important than e.g. the one from the error response?

Reply:
I will change it to debug one